llvm/include/llvm/Transforms/Utils/SampleProfileInference.h

```diff
@@ -59,6 +59,8 @@ struct FlowFunction {
   /// The index of the entry block.
   uint64_t Entry{0};
   uint64_t Sink{UINT64_MAX};
+  // Matched execution count for the function.
+  uint64_t MatchedExecCount{0};
```

Comment:
nit: I'd be careful about adding this to `FlowFunction` -- strictly speaking this doesn't belong to flow function, which just describe the CFG and if we add function level "attributes" to flow functions, we'd have a lot more here.